jog #3 - recurring tasks

graceleeLACRM opened the pull request:
- user can select if task is recurring and what days it recurs if so
- recurring/non-recurring tasks are inserted accordingly in separate tables (but if the user adds a recurring task mid-week, the task will be added once to the non-recurring table so it's displayed the same week it was added)
- recurring tasks are added to daily tasks table once a week by storing and updating the week # of the last insertion for the appropriate user
- user can view/delete recurring tasks

graceleeLACRM committed:
jog #3 - recurring tasks

- user can select if task is recurring and what days it recurs if so
- recurring/non-recurring tasks are inserted accordingly in separate tables (but if the user adds a recurring task mid-week, the task will be added once to the non-recurring table so it's displayed the same week it was added)
- recurring tasks are added to daily tasks table once a week by storing and updating the week # of the last insertion for the appropriate user
- user can view/delete recurring tasks

renodubois reviewed (changes requested):
a few little things!

graceleeLACRM committed:
jog #3 - recurring tasks

- changed add_task.php to return an object rather than array
- properly named variables
- used template literal string

Regarding your comment about the checkbox variable, I'm having trouble calling it in a different function from where it was initialized (the error says that the checkbox variable was not defined). I tried setting it as a global variable on the first line of code, outside of any function, but when I do that it seems to return null when the variable is used in any of my functions.

I just initialized the variable in every function that I used it in. Is there a way to fix the errors I mentioned above?
